src/commands/optimize.ts

```diff
@@ -84,21 +84,12 @@ export default class Optimize extends Command {
     this.optimizations = flags.optimization as Optimizations[];
     this.outputMethod = flags.output as Outputs;
 
-    this.metricsMetadata.optimized = true;
-
-    if (report.moveToComponents?.length) {
-      this.metricsMetadata.optimization_moveToComponents = true;
-    }
-    if (report.removeComponents?.length) {
-      this.metricsMetadata.optimization_removeComponents = true;
-    }
-    if (report.reuseComponents?.length) {
-      this.metricsMetadata.optimization_reuseComponents = true;
-    }
-
-    if (!(report.moveToComponents?.length || report.removeComponents?.length || report.reuseComponents?.length)) {
-      this.log(`No optimization has been applied since ${this.specFile.getFilePath() ?? this.specFile.getFileURL()} looks optimized!`);
+    if (report.moveToComponents?.length || report.removeComponents?.length || report.reuseComponents?.length) {
```

Comment:
You don't need to an else if you keep the old behavior and initialize the value as `false` by default (or even the opposite). 

```ts
this.metricsMetadata.optimized = false;
if (!(report.moveToComponents?.length || report.removeComponents?.length || report.reuseComponents?.length)) {
      this.log(`No optimization has been applied since ${this.specFile.getFilePath() ?? this.specFile.getFileURL()} looks optimized!`);
      return;
}

this.metricsMetadata.optimized = true;
```